internal/topic/topicwriterinternal/message.go

```diff
@@ -23,7 +23,6 @@ type PublicMessage struct {
 	// partitioning at level message available by protocol, but doesn't available by current server implementation
 	// the field hidden from public access for prevent runtime errors.
 	// it will be published after implementation on server side.
-	futurePartitioning PublicFuturePartitioning
```

Comment:
This line was removed because the structcheck linter wrote `internal/topic/topicwriterinternal/message.go:26:2: "futurePartitioning" is unused (structcheck)
        futurePartitioning PublicFuturePartitioning
`